dev/io.openliberty.jakarta.mail.2.0/bnd.bnd

```diff
@@ -0,0 +1,83 @@
+#*******************************************************************************
+# Copyright (c) 2017, 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+
+bVersion=2.0
+
+
+bSymbolicName=io.openliberty.jakarta.mail.2.0
+
+Bundle-Name: Jakarta Mail 2.0 API
```

Comment:
I think this description is wrong? This bundle exports an Open Liberty API, not a Jakarta one.